geomstats/geometry/discrete_curves.py

```diff
@@ -325,6 +326,48 @@ def square_root_velocity_inverse(self, srv, starting_point):
 
         return curve
 
+    def srv_inner_product(self, srv_1, srv_2):
```

Comment:
Sorry, I forget. Remind me how this function is different from `SRVMetric.inner_product` and from `SRVMetric.pointwise_inner_product`?

Reply:
`SRVMetric.pointwise_inner_product` takes two tangent vectors [u_1, u_2, ...] and [v_1, v_2, ...] at a base curve and returns an array with the inner products (`ambient_metric.inner_product`) between u_1 and v_1, u_2 and v_2 etc. `SRVMetric.inner_product` is inherited from `RiemannianMetric` and is supposed to compute the inner_product between two tangent vectors at a base point by using `RiemannianMetric.metric_matrix`, which is not implemented (by default and in `SRVMetric`). `SRVMetric.srv_inner_product` takes two srv representations and computes their L² inner product since the elastic metric becomes the L² metric in the srv space.